drivers/oled/oled_driver.h

```diff
@@ -41,14 +39,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   // If OLED_BLOCK_TYPE is uint8_t, these tables would look like:
   // #define OLED_SOURCE_MAP { 0, 8, 16, 24, 32, 40, 48, 56, 64, 72, 80, 88, 96, 104, 112, 120 }
   // #define OLED_TARGET_MAP { 56, 120, 48, 112, 40, 104, 32, 96, 24, 88, 16, 80, 8, 72, 0, 64 }
+
+  // the column address corresponding to the first column in the display hardware
+  #define OLED_COLUMN_OFFSET 0
 #else // defined(OLED_DISPLAY_128X64)
   // Default 128x32
   #define OLED_DISPLAY_WIDTH 128
   #define OLED_DISPLAY_HEIGHT 32
   #define OLED_MATRIX_SIZE (OLED_DISPLAY_HEIGHT / 8 * OLED_DISPLAY_WIDTH) // 512 (compile time mathed)
   #define OLED_BLOCK_TYPE uint16_t // Type to use for segmenting the oled display for smart rendering, use unsigned types only
-  #define OLED_BLOCK_COUNT (sizeof(OLED_BLOCK_TYPE) * 8) // 16 (compile time mathed)
-  #define OLED_BLOCK_SIZE (OLED_MATRIX_SIZE / OLED_BLOCK_COUNT) // 32 (compile time mathed)
```

Comment:
Didn't realize I duplicated this also in the .c file. Good catch on removal. Please remove this from the docs as well.